tss-esapi/tests/integration_tests/error_tests/return_code_tests/tpm_tests/tpm_format_zero_warning_tests.rs

```diff
@@ -60,12 +60,26 @@ macro_rules! test_valid_conversion {
             expected_tss_rc,
             actual_rc.into(),
             "TpmFormatZeroResponseCode with {} did not convert into expected {} TSS2_RC in the TPM layer.",
-            std::stringify!(TpmFormatZeroError::$item),
+            std::stringify!(TpmFormatZeroWarning::$item),
```

Comment:
:thinking: it's interesting that you've changed Error to Warning here... how did it work previously if they have different enum variants?

Reply:
> 🤔 it's interesting that you've changed Error to Warning here... how did it work previously if they have different enum variants?

I think it does some fancy tricks to never treat what is written there as type but just as text and there for it worked.